scripts/fiber/tests-failing.txt

```diff
@@ -194,4 +153,5 @@ src/renderers/shared/stack/reconciler/__tests__/refs-test.js
 * Should increase refs with an increase in divs
 
 src/test/__tests__/ReactTestUtils-test.js
+* traverses children in the correct order
```

Comment:
This commit actually fixes some issues so some tests get further but not enough to complete them so it looks like a pure regression.

We need to fix findAllInRenderedChildren and findDOMNode with a different strategy somehow asap. I don't know how yet. Any ideas?